src/index.ts

```diff
@@ -5,3 +5,71 @@ export { applyFilter } from "./filter";
 export { fromPandoc, toPandoc } from "./pandoc";
 export { renderDjot } from "./djot-renderer";
 export { version } from "./version";
+export { isBlock, isInline } from "./ast";
```

Comment:
Didnt export `IsRow, IsCaption` in the light of https://github.com/jgm/djot.js/issues/57